Tools/AP_Periph/Parameters.cpp

```diff
@@ -373,6 +373,13 @@ const AP_Param::Info AP_Periph_FW::var_info[] = {
     // @RebootRequired: True
     GSCALAR(esc_pwm_type, "ESC_PWM_TYPE",     0),
 
+    // @Param: ESC_CMD_TIMO
+    // @DisplayName: ESC Command Timeout
+    // @Description: This is the duration (ms) with which to hold the last driven ESC command before timing them out and zeroing outputs. To disable zeroing of outputs in event of CAN loss, use 0. Use values greater than the expected duration between two CAN frames to ensure Periph is not starved of ESC Raw Commands
+    // @Range: 0 10000
+    // @User: Advanced
+    GSCALAR(esc_command_timeout_ms, "ESC_CMD_TIMO", 50),
```

Comment:
Should we make the timeout in seconds rather than milliseconds?  Internally we could convert the secs to ms but user facing might be better in seconds.

Reply:
Removing the parameter.